### PR TITLE
Add nodes required by MCUboot

### PR DESCRIPTION
**Pull request by michalek-no**

### CODE REVIEW

**Review by sigvartmh — APPROVED** [review]
LGTM You just need to fix git tags.

**Review by nvlsianpu — APPROVED** [review]

**Comment by gchwier** [comment]
Please update commit messages with the link to upstream PRs and open PR on sdk-nrf with manifest update

**Review by gchwier — APPROVED** [review]

File: Kconfig.zephyr [hunk 1]
```diff
@@ -129,6 +129,7 @@ endif # HAS_FLASH_LOAD_OFFSET
 config ROM_START_OFFSET
 	hex
 	prompt "ROM start offset" if !BOOTLOADER_MCUBOOT
+	default 0x800 if BOOTLOADER_MCUBOOT && SOC_PLATFORM_NRF54L
```
Comment (jfischer-no) [inline Kconfig.zephyr:132]: https://github.com/zephyrproject-rtos/zephyr/pull/68393/files/8e0adeb8434595c29a8960b36d62d2c4c6efab1f#r1492956308, Seems like a valid complaint, right?

**Comment by gchwier** [comment]
@michalek-no can you also add third commit: https://github.com/zephyrproject-rtos/zephyr/pull/68393/commits/580fb8479b9785ea51c655671ca0bc03ab061d72
this is also added in upstream PR: https://github.com/zephyrproject-rtos/zephyr/pull/68393

**Review by de-nordic — APPROVED** [review]

**Review by SeppoTakalo — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

**Pull request merged**